imod_coupler/drivers/ribametamod/config.py

```diff
@@ -9,8 +9,8 @@
 
 class Kernels(BaseModel):
     modflow6: Modflow6
-    ribasim: Ribasim
-    metaswap: Metaswap
+    ribasim: Optional[Ribasim]
+    metaswap: Optional[Metaswap]
```

Comment:
```suggestion
    ribasim: Ribasim | None = None
    metaswap: Metaswap | None = None
```

If you don't set `=None` the key will still be required